### DISCUSSION TITLE
How can I limit the history of Zellij?

### DISCUSSION THREAD

**Discussion by thisismygitrepo** [zellij-org/zellij#2781] [Q&A] [answered]
My program prints indefinitely to the terminal for weeks, and memory seems to go up.
I'm not sure if where the leak is, but I'm just wondering now, is there a way to limit how much Zellij keeps from the stuff printed to the terminal?

**Comment by Lockszmith-GH** [comment, marked as answer]
According to the [documentation](https://zellij.dev/documentation/options.html?highlight=back#scroll_buffer_size) the scroll buffer is set to 10,000 lines - so it's not unlimited - shouldn't affect memory like that.

Have you updated zellij recently? There was [a memory leaks](https://github.com/zellij-org/zellij/issues/1625#ref-pullrequest-1836268716) fixed about a month ago.
